src/Peering/Peering/help/Get-AzLegacyPeering.md

```diff
@@ -58,7 +58,7 @@ Gets the legacy peering for Seattle
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
```

Comment:
@rileymckenna it looks like the help was generated without the full type names. These are included if you run the `Update-MarkdownHelpModule` command with the `-UseFullTypeName` parameter